api/turing/validation/validator_test.go

```diff
@@ -287,15 +301,22 @@ type routerConfigTestCase struct {
 	trafficRules       models.TrafficRules
 	autoscalingPolicy  *models.AutoscalingPolicy
 	expectedError      string
+	logConfig          *request.LogConfig
 }
 
-func (tt routerConfigTestCase) RouterConfig(protocol routerConfig.Protocol) *request.RouterConfig {
```

Comment:
no idea why protocol is in method parameter (written by me 😅  previously i think), removed and refer to input args instead for consistency